marketplacetests/firefox_accounts/app.py

```diff
@@ -37,6 +44,20 @@ def login(self, email, password):
         self.type_password(password)
         self.tap_sign_in()
 
+    def switch_to_in_app_login_frame(self):
+        self.marionette.switch_to_frame()
+        self.wait_for_element_present(*self._in_app_login_frame_locator)
+        login_iframe = self.marionette.find_element(*self._in_app_login_frame_locator)
+        self.marionette.switch_to_frame(login_iframe)
+
+    def switch_to_fxa_login_frame(self):
+        self.marionette.switch_to_frame()
+        self.wait_for_element_present(*self._ff_accounts_frame_locator)
+        self.frame = self.marionette.find_element(*self._ff_accounts_frame_locator)
+        self.marionette.switch_to_frame(self.frame)
```

Comment:
These four lines of code (54-57) are basically the same as the ones in `switch_to_in_app_login_frame` (lines 48-51), with the only difference being the locator used. I suggest making them a single method that accepts a locator as an argument, and then that method can be called when needed.